### ISSUE TITLE
Bad removal of a function

### ISSUE THREAD

**Issue by Tyler-Keith-Thompson** [nicklockwood/SwiftFormat#1776]
I'm writing a vapor application and have this code:
```swift
struct OpenAPIController: RouteCollection {
    let info = InfoObject(title: "Swagger {{cookiecutter.service_name}} - OpenAPI",
                          description: "{{cookiecutter.description}}",
                          contact: .init(email: "{{cookiecutter.email}}"),
                          version: Version(0, 0, 1))

    func boot(routes: RoutesBuilder) throws {
        routes.get("swagger", "swagger.json") {
            $0.application.routes.openAPI(info: info)
        }
        .excludeFromOpenAPI()
    }

}
```

SwitFormat (latest version) oddly cuts out a huge chunk of the code:
```swift
struct OpenAPIController: RouteCollection {
    let info = InfoObject(title: "Swagger {{cookiecutter.service_name}} - OpenAPI",
                          description: "{{cookiecutter.description}}",
                          contact: .init(email: "{{cookiecutter.email}}"),
                          version: Version(0, 0, 1)) throws {
        routes.get("swagger", "swagger.json") {
            $0.application.routes.openAPI(info: info)
        }
        .excludeFromOpenAPI()
    }
}
```

Which notably doesn't compile. My .swiftformat config is:
```yaml
--acronyms ID,URL,UUID
--allman false
--assetliterals visual-width
--asynccapturing 
--beforemarks 
--binarygrouping none
--categorymark "MARK: %c"
--classthreshold 0
--closingparen same-line
--closurevoid remove
--commas always
--conflictmarkers reject
--decimalgrouping none
--elseposition same-line
--emptybraces spaced
--enumnamespaces always
--enumthreshold 0
--exponentcase lowercase
--exponentgrouping disabled
--extensionacl on-declarations
--extensionlength 0
--extensionmark "MARK: - %t + %c"
--fractiongrouping disabled
--fragment false
--funcattributes preserve
--generictypes 
--groupedextension "MARK: %c"
--guardelse same-line
--header ignore
--hexgrouping none
--hexliteralcase uppercase
--ifdef indent
--importgrouping testable-last
--indent 4
--indentcase true
--indentstrings false
--lifecycle 
--lineaftermarks true
--linebreaks lf
--markcategories true
--markextensions always
--marktypes always
--maxwidth none
--modifierorder 
--nevertrailing 
--nospaceoperators 
--nowrapoperators 
--octalgrouping none
--operatorfunc spaced
--organizetypes actor,class,enum,struct
--patternlet inline
--ranges spaced
--redundanttype infer-locals-only
--self remove
--selfrequired 
--semicolons inline
--shortoptionals always
--smarttabs enabled
--someany true
--stripunusedargs always
--structthreshold 0
--tabwidth unspecified
--throwcapturing 
--trailingclosures 
--trimwhitespace always
--typeattributes preserve
--typeblanklines remove
--typemark "MARK: - %t"
--varattributes preserve
--voidtype void
--wraparguments after-first
--wrapcollections before-first
--wrapconditions preserve
--wrapeffects never
--wrapenumcases always
--wrapparameters after-first
--wrapreturntype preserve
--wrapternary default
--wraptypealiases preserve
--xcodeindentation disabled
--yodaswap always
--disable strongOutlets,wrap,wrapMultilineStatementBraces
--enable blankLineAfterImports,wrapSwitchCases
```

**Comment by calda** [comment]
Thanks for the report, here's a fix: https://github.com/nicklockwood/SwiftFormat/pull/1778

**Comment by nicklockwood** [comment]
@Tyler-Keith-Thompson fixed in 0.54.3
